src/main/java/org/jitsi/srtp/crypto/Aes.java

```diff
@@ -282,6 +281,14 @@ private static CipherFactory benchmark(
             if (factory == null)
                 continue;
 
+            // The user may have specified a specific CipherFactory class
+            // (name) through setFactoryClassName(String), Practically, FACTORY_CLASS_NAME may override
+            // minFactory and, consequently, it may appear that the benchmark is
+            // unnecessary. Technically though, the specified CipherFactory may
+            // malfunction. That is why FACTORY_CLASS_NAME is selected after it has
+            // proven itself functional.
+            boolean chosenFactoryClass = (factory.getClass().equals(Aes.factoryClass));
```

Comment:
Extra parens on purpose?

Reply:
Ah, no, I reorganized this from an `if` statement.